src/main/java/org/btcprivate/wallets/fullnode/ui/SendCashPanel.java

```diff
@@ -405,10 +405,10 @@ private void sendCash()
         
         String errorMessage = null;
 
-        int sourceAddressProperLength = 35;
-		if (sourceAddress.startsWith("zk")) sourceAddressProperLength = 95;
-		int destinationAddressProperLength = 35;
-		if (destinationAddress.startsWith("zk")) destinationAddressProperLength = 95;
+		final int BADDRESSPROPERLENGTH = 35;
```

Comment:
maybe `B_ADDRESS_PROPER_LENGTH` for style

Reply:
Agreed and done.